packages/docs/src/page-configs/ui-elements/select/examples/Variations.vue

```diff
@@ -19,7 +19,7 @@
       :options="options"
       v-model="valueMultipleMax"
       multiple
-      max-selections=2
+      :max-selections="2"
```

Comment:
Would be better to change maxSelections prop type to `Number | String`. End user can face the same problem there, if he wants to simply set `max-selections="2"`. 

```suggestion
      max-selections="2"
```

Reply:
fixed it